stl/inc/numeric

```diff
@@ -646,22 +642,12 @@ _NODISCARD constexpr common_type_t<_Mt, _Nt> lcm(const _Mt _Mx, const _Nt _Nx) n
 
 #if _HAS_CXX20
 template <class _Flt>
-constexpr _Flt _Floating_max{};
-template <>
-inline constexpr float _Floating_max<float> = FLT_MAX;
-template <>
-inline constexpr double _Floating_max<double> = DBL_MAX;
-template <>
-inline constexpr long double _Floating_max<long double> = LDBL_MAX;
+constexpr _Flt _Floating_max = _STD _Bit_cast<_Flt>(
+    static_cast<_Floating_type_traits<_Flt>::_Uint_type>(_Floating_type_traits<_Flt>::_Shifted_exponent_mask - 1));

```

Comment:
I think this is unnecessarily complicated, and there's an easier way to move it into `_Floating_type_traits` itself, where the minimum and maximum values are definitely something that's natural to provide.